pom.xml

```diff
@@ -39,13 +39,13 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.build.targetJdk>1.7</project.build.targetJdk>
+        <project.build.targetJdk>1.8</project.build.targetJdk>
         <shadeBase>io.trino.hive.jdbc.\$internal</shadeBase>
-        <dep.slf4j.version>1.6.6</dep.slf4j.version>
-        <dep.hadoop.version>0.20.2</dep.hadoop.version>
-        <dep.hive.version>0.13.1</dep.hive.version>
-        <dep.thrift.version>0.12.0</dep.thrift.version>
-        <dep.guava.version>11.0.2</dep.guava.version>
+        <dep.slf4j.version>1.7.10</dep.slf4j.version>
+        <dep.hadoop.version>1.2.1</dep.hadoop.version>
+        <dep.hive.version>3.1.3</dep.hive.version>
```

Comment:
Update the project version to `3.1.3-1-SNAPSHOT` so that it matches the Hive version.

Update the README to use the new version, or simply remove the Hive version from the README.